sdk/eidolon_ai_sdk/cpu/llm/mistral_llm_unit.py

```diff
@@ -117,15 +116,8 @@ async def convert_to_mistral(message: LLMMessage):
         raise ValueError(f"Unknown message type {message.type}")
 
 
-class MistralModelEnum(str, Enum):
-    MistralTiny = "mistral-tiny-latest"
-    MistralSmall = "mistral-small-latest"
-    MistralMedium = "mistral-medium-latest"
-    MistralLarge = "mistral-large-latest"
-
-
 class MistralGPTSpec(BaseModel):
-    model: MistralModelEnum = Field(default=MistralModelEnum.MistralMedium, description="The model to use for the LLM.")
+    model: str = Field(default="mistral-large-latest", description="The model to use for the LLM.")
```

Comment:
Can you change this back to a literal of supported types or add fallback logic on the model selection? (line 240)